packages/evm/src/libs/commonUtils.ts

```diff
@@ -14,3 +14,13 @@ export const collectionIdToERC1155Address = (collectionId: string | number): str
 	const collectionIdInHex = (+collectionId).toString(16).padStart(8, "0");
 	return getAddress(`0xBBBBBBBB${collectionIdInHex.toUpperCase()}000000000000000000000000`);
 };
+
+export const contractAddressToNativeId = (contractAddress: `0x${string}`): number | null => {
+	if (!isAddress(contractAddress)) throw new Error("contractAddress is not a valid address");
+	const startPrefix = contractAddress?.substring(0, 10);
+	const allowed = ["0xaaaaaaaa", "0xcccccccc", "0xbbbbbbbb"];
```

Comment:
nit: why not alphabetical list; i.e.

```suggestion
	const allowed = ["0xaaaaaaaa", "0xbbbbbbbb", "0xcccccccc"];
```

Reply:
✅